tests/conftest.py

```diff
@@ -33,3 +40,46 @@ def uvicorn_server():
     time.sleep(0.3)
     yield f"http://localhost:{port}"
     proc.kill()
+
+
+@pytest.fixture
```

Comment:
Here I am just moving these fixtures into `conftest.py` so we can use them in multiple files.